onnx/test/test_external_data.py

```diff
@@ -283,16 +283,6 @@ def create_test_model_both(self):  # type: () -> Tuple[Text, ModelProto]
             model_file.write(model.SerializeToString())
         return model_filename, model
 
-    # 2GB models would fail onnx.checker with ModelProto but pass with model path
```

Comment:
Remove this in this PR to pass pytest. It will be removed by https://github.com/onnx/onnx/pull/3048.